index.html

```diff
@@ -45,5 +45,194 @@ <h1 class="title">I'm Mariane <br>Glad to see you!</h1>
+
+                <ul class="languages">
+                    <li><button class="lang" id="lan-1">html</button></li>
+                    <li><button class="lang" id="lan-2">css</button></li>
+                    <li><button class="lang" id="lan-3">javaScript</li></button></li>
+                </ul>
+
+                <button class="see" id="see-1">See Project</button>
+            </div>
+        </div>
+
+        <div class="works works2">
+            <img src="images/Snapshoot Portfolio (3).svg" alt="" class="tonic">
+            <img src="images/Snapshoot Portfolio4.svg" alt="" class="tonic-1">
+            <div>
+                <h1 class="stories">Multi-Post Stories</h1>
+
+                <ul class="description">
+                    <li class="canopy">CANOPY</li>
+                    <li><img src="images/Counter.svg" alt="" id="counter"></li>
+                    <li class="back">Back End Dev</li>
+                    <li><img src="images/Counter.svg" alt="" id="counter"></li>
+                    <li class="back">2015</li>
+                </ul>
+
+                <p class="more-info">A daily selection of privately 
+                    personalized reads; no accounts or sign-ups required.
+                </p>
+
+                <ul class="languages">
+                    <li><button class="lang" id="lan-1">html</button></li>
+                    <li><button class="lang" id="lan-2">css</button></li>
+                    <li><button class="lang" id="lan-3">javaScript</li></button></li>
+                </ul>
+
+                <button class="see" id="see-1">See Project</button>
+            </div>
+        </div>
+    </section>
+
+    <section class="about-myself" id="about-myself">
+        <div class="myself">
+            <div>
+                <h1 class="about">About Myself</h1>
+                <p class="experience">Hello I'm a software developer! 
+                        I can help you build a product , feature or website Look 
+                        through some of my work and experience! If you like what you 
+                        see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+
+                <p class="connect-1">LET'S CONNECT</p>
+
+                <ul class="icons-1">
+                    <li><img src="images/Vector (4).svg" class="icons-2"></li>
+                    <li><img src="images/Group.svg" class="icons-2"></li>
+                    <li><img src="images/Vector (5).svg" class="icons-2"></li>
+                    <li><img src="images/Vector.svg" class="icons-2"></li>
+                    <li><img src="images/Vector (3).svg" class="icons-2"></li>
+                </ul>
+
+                <button class="resume" id="resume-1">Get my resume</button>
+
+            </div>
+            
+            <div class="language-container">
+
+                    <ul class="languages-below">
+                        <li class="languages-1" id="languages-23">Languages</li>
+
```

Comment:
- [ ] In the [figma design ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=zjF2gQCiVeUiBNY5-0) there is a drop down arrow next to the language heading, and as we are trying to aim for a pixel perfect design I would suggest you to add this arrow icon


| Figma design | Your design |
|--------------|--------------|
|<img width="295" alt="Screenshot 2023-04-29 at 5 36 26 PM" src="https://user-images.githubusercontent.com/44798044/235303001-3216e126-d1f6-4f0c-8765-a4a5246c7849.png">|<img width="287" alt="Screenshot 2023-04-29 at 5 29 44 PM" src="https://user-images.githubusercontent.com/44798044/235303026-6e82284a-5ef5-4aca-8198-934808ab882c.png">|

Reply:
Added the arrow icon